db/migrate/20220913130404_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Please add foreign keys to create one-to-many relationships between users and comments, likes and posts. You can check out how to do this [here](https://guides.rubyonrails.org/active_record_migrations.html#references). For example, when relating posts with users, run something similar to this in the terminal 
`rails generate migration AddUserRefToPosts user:references`